config/k8s/config.yaml

```diff
@@ -1,9 +1,9 @@
-sharedSecretFile: $(CONFIG_MOUNT_PATH)/shared-secret
-baseUrl: $(OAUTH_HOST)
+sharedSecret: yaddayadda123$@#**
```

Comment:
Is this dummy value required? Wouldn't something like `<REPLACE_ME_WITH_A_REAL_VALUE>` be more descriptive?

Reply:
yes, replaced with something that can be used lated as placeholder